modules/loader/src/test/scala/com/snowplowanalytics/snowplow/storage/bigquery/loader/LoaderSpec.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2018-2021 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.storage.bigquery.loader
+
+import com.spotify.scio.testing.PipelineSpec
+
+class LoaderSpec extends PipelineSpec {
+  "Loader" should "split inputs" in {
+    // TODO
```

Comment:
?

Reply:
We can't do this test without a real-world BQ table to write to. Last time we tried, it couldn't be mocked. It's definitely on the to-do list, just need to figure out how the table will be created / maintained.